src/core/events/dispatcher.ts

```diff
@@ -21,18 +22,33 @@ export default class Dispatcher {
     return this;
   }
 
-  bind_all(callback : Function) {
+  bind_global(callback : Function) {
     this.global_callbacks.push(callback);
     return this;
   }
 
-  unbind(eventName : string, callback : Function, context?: any) {
+  unbind(eventName? : string, callback? : Function, context?: any) {
```

Comment:
This API is a bit dangerous, since arguments passed to the function could resolve to undefined (to due a bug in a client's use of the library) and then that would result in unbinding all listeners. However, that should not be part of this PR.